packages/babel-types/test/fields.js

```diff
@@ -22,6 +22,8 @@ const ignoredFields = {
   TSDeclareMethod: { id: true },
   ...(process.env.BABEL_8_BREAKING
     ? {
+        ClassExpression: { superTypeParameters: true },
+        ClassDeclaration: { superTypeParameters: true },
```

Comment:
I will extract https://github.com/babel/babel/pull/16979/commits/763ebaaa6198bec50fc81b84e1aa1e0bd10738a8 into a new PR so that we don't have to take care of Babel 7/8 AST changes in this test.